README.md

```diff
@@ -1,2 +1,41 @@
 # AP_IMRT_Schema
 Contains SQL schema migrations to support the IMRT application(s)
+
+Uses the flyway gradle plugin: https://flywaydb.org/documentation/gradle/
+
+Assumes that the database itself has already been created and configured
+
+To use, configuir the following properties, either via environment variables, command line arguments, are gradle.properties:
+<pre>
+flyway.user=<user>
+flyway.password=<password>
+flyway.url=<url> 
+Sample URL for local database named imrt: jdbc:postgresql://localhost:5432/imrt
+</pre>
+
+Then run the flyway tasks, for example:
+<pre>
+./gradlew flywayClean
+./gradlew flywayMigrate
+</pre>
+
+## SQL Conventions
+
+SQL files should conform to the following conventions:
+* Use all lowercase letters for names
+* Separate words using underscores
+* Foreign key names should follow the format \<fk table>_\<fk column>
+* Timestamp field names should end is _at, for example 'created_at'
+* Timestamp columns should use the timestamptz type
+* Columns denoting authorship should end in _by, for example 'updated_by'
```

Comment:
Can you add a bullet describing the audit fields that should be present in each table?